.vscode/launch.json

```diff
@@ -24,7 +24,33 @@
           "--runInBand",
           "--config",
           "${workspaceRoot}/packages/beachball/jest.config.js",
-          "${fileBasename}"
+          "${fileBasenameNoExtension}"
+        ]
+      },
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Jest E2E Tests",
```

Comment:
Let's update the task above with `"name": "Debug Jest Unit Tests"`

Reply:
yes!